src/index.js

```diff
@@ -1,6 +1,4 @@
-import fontawesome from '@fortawesome/fontawesome'
-import FontAwesomeIcon from './components/FontAwesomeIcon'
+import { noAuto } from '@fortawesome/fontawesome'
+export { FontAwesomeIcon } from './components/FontAwesomeIcon'
```

Comment:
Can we make this `export default as FontAwesomeIcon from './components/FontAwesomeIcon'`?